app/screens/edit_profile/components/form.tsx

```diff
@@ -72,16 +74,18 @@ const styles = StyleSheet.create({
 const ProfileForm = ({
     canSave, currentUser, isTablet,
     lockedFirstName, lockedLastName, lockedNickname, lockedPosition,
-    onUpdateField, userInfo, submitUser,
+    onUpdateField, userInfo, submitUser, error,
 }: Props) => {
     const theme = useTheme();
-    const {formatMessage} = useIntl();
+    const intl = useIntl();
     const firstNameRef = useRef<FloatingTextInputRef>(null);
     const lastNameRef = useRef<FloatingTextInputRef>(null);
     const usernameRef = useRef<FloatingTextInputRef>(null);
     const emailRef = useRef<FloatingTextInputRef>(null);
     const nicknameRef = useRef<FloatingTextInputRef>(null);
     const positionRef = useRef<FloatingTextInputRef>(null);
+    const errorMessage = getErrorMessage(error, intl) as string;
```

Comment:
I think it makes more sense to do here something like `error == null ? undefined : getErrorMessage(error, intl) as string`. That way, the change in line 194 is a bit less cryptic.

Notice that `== null` also checks for undefined, in constrast to `=== null`.